### PR TITLE
Fix scope splitting

### PR DESCRIPTION
**Pull request by aptinio**

### CODE REVIEW

**Comment by yordis** [comment]
Hey there, thank you so much for the contribution. Do you mind update https://github.com/ueberauth/ueberauth_google/blob/master/CHANGELOG.md and bump the version in the `mix.exs` so we could release it?

**Comment by aptinio** [comment]
Don't mind at all. Thanks @yordis!

**Review by yordis — APPROVED** [review]

File: CHANGELOG.md
Comment (aptinio) [inline CHANGELOG.md:1]: @yordis, I didn't realize until now that my editor formatted this file (using Prettier). Sorry about that :sweat_smile:

Comment (yordis) [reply CHANGELOG.md:1]: All good, it is not your fault. I need to put Actions in place that helps with these things honestly

**Pull request merged**